pages/order.vue

```diff
@@ -98,28 +147,64 @@
         return getPlan(this.$auth.user.plan_id)
       },
       currentPrice(){
-        return this.plan[this.planTerm].currentPrice
+        return this.plan[this.planTerm].prices[this.currentUsersRangeKey]
+      },
+      totalPrice() {
+        return this.currentPrice * this.seats
       },
       yearlyPrice(){
-        return this.plan['year'].currentPrice
+        return this.plan['year'].prices[this.currentUsersRangeKey]
       },
       monthlyPrice(){
-        return this.plan['month'].currentPrice
+        return this.plan['month'].prices[this.currentUsersRangeKey]
+      },
+      teamPurchaseMaxSavings() {
+        let teamCostDifference = Math.floor(this.plan[this.planTerm].prices["5+"] / this.plan[this.planTerm].prices["1"]  * 100)
```

Comment:
I think we could do a `prices` computed property defined as follows:

```
return this.plan[this.planTerm].prices
```

The pattern is used 4 times already, and would make teamSavings and teamPurchaseMaxSavings more legible.